ReduxTestTask/Models/NetworkAtricle.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  NetworkAtricle.swift
+//  TestTask
+//
+//  Created by Polina on 8/18/19.
+//  Copyright © 2019 Polina. All rights reserved.
+//
+
+import Foundation
+
+struct NetworkResponse: Decodable {
```

Comment:
it's tightly coupled with NetworkArticle, so NetworkResponse isn't the best name for this structure